airbyte-integrations/connectors/source-faker/setup.py

```diff
@@ -19,7 +19,7 @@
     author_email="evan@airbyte.io",
     packages=find_packages(),
     install_requires=MAIN_REQUIREMENTS,
-    package_data={"": ["*.json"]},
+    package_data={"": ["*.json", "schemas/*.json", "record_data/*.json"]},
```

Comment:
For all connectors which might one day need to do this... is there a globpattern we should add to bundle all data files like `["**/*.json", "**/*.yaml"]`?